src/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -103,11 +103,24 @@ protected virtual JObject GetBlockHeader(JArray _params)
         [RpcMethod]
         protected virtual JObject GetContractState(JArray _params)
         {
-            UInt160 script_hash = UInt160.Parse(_params[0].AsString());
+            UInt160 script_hash = UInt160.Parse(ToScriptHash(_params[0].AsString()));
             ContractState contract = Blockchain.Singleton.View.Contracts.TryGet(script_hash);
             return contract?.ToJson() ?? throw new RpcException(-100, "Unknown contract");
         }
 
+        public static string ToScriptHash(string addressOrScriptHash)
+        {
+            foreach (var native in NativeContract.Contracts)
+            {
+                if (addressOrScriptHash.Equals(native.Name, StringComparison.InvariantCultureIgnoreCase) ||
+                    addressOrScriptHash == native.Id.ToString())
```

Comment:
https://github.com/neo-project/neo-modules/pull/340#issuecomment-687749790

Reply:
We only searched for native contracts, not other contracts. There's no harm in doing this, right?

Reply:
SDK is not a solution, RPC client is used in many scenarios, not just for software development. And we'd like to have it as little data as possible to function, that's why it's important to have the server replying to such requests. This particular change allows to remove native contract hashes from the client.

Reply:
Can we also add an ability to get non-native contracts hashes by ID here?

Reply:
Only native contracts can be searched by ID, not other contracts.  Because other contracts are stored as key-value pairs, they cannot be traversed. @annashaleva